wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -502,9 +504,14 @@ public static void updateBeanValue(final WComponent component, final boolean vis
 		}
 
 		if (component instanceof Container) {
+
+			final List<WComponent> children = ((Container) component).getChildren();
+
+			Collections.reverse(children);
```

Comment:
Invoking reverse on an unmodifiable List will probably throw an exception. Might need to add the children to a new List before reversing.